web-api/src/storage.rs

```diff
@@ -66,7 +67,12 @@ pub(crate) enum DeletionError {
 
 #[async_trait]
 pub(crate) trait Document {
-    async fn get_by_ids(&self, ids: &[&DocumentId]) -> Result<Vec<PersonalizedDocument>, Error>;
+    async fn get_interacted(&self, ids: &[&DocumentId]) -> Result<Vec<InteractedDocument>, Error>;
+
+    async fn get_personalized(
```

Comment:
:art: We can keep the naming, but I'm not sure if it's most appropriate as they do get used in other contexts.

Reply:
i agree that the names aren't perfect, i just could come up with any better, i'm happy to change them for new suggestions.